### DISCUSSION TITLE
Animation in Paraview is too quick

### DISCUSSION THREAD

**Discussion by jamesclooney365** [idaholab/moose#22781] [Q&A General] [answered]
Dear MOOSE Developers,
I have generated the exodus file for Porousflow tutorial11 but having an issue with how it plays in Paraview. The file plays okay but very fast compared to the very slow simulation/animation that is described on the moose webpage. I tried using the real-time mode and changing the iteration time length, but it didn’t work. Here is my file: Any help or guidance? Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You may run more time steps to have a slower video, or you can change the animation parameter of Paraview
https://discourse.paraview.org/t/how-to-change-frame-rate/3623 (this is what you tried, but it should work)

The file did not get attached to the post btw

Guillaume

**Reply by jamesclooney365** [reply]
It worked! thanks!
